project/GenTypeClass.scala

```diff
@@ -35,16 +35,11 @@ object TypeClass {
 
   lazy val invariantFunctor = TypeClass("InvariantFunctor", *->*)
   lazy val functor = TypeClass("Functor", *->*, extendsList = Seq(invariantFunctor))
-  lazy val applyDivide = TypeClass("ApplyDivide", *->*, extendsList = Seq(invariantFunctor))
-  lazy val applicativeDivisible = TypeClass("ApplicativeDivisible", *->*, extendsList = Seq(applyDivide))
-  lazy val derives = TypeClass("Derives", *->*, extendsList = Seq(coapplicativeCodivide, applicativeDivisible))
-  lazy val coapplicativeCodivide = TypeClass("CoapplicativeCodivide", *->*, extendsList = Seq())
-  lazy val codivide = TypeClass("Codivide", *->*, extendsList = Seq(coapplicativeCodivide))
-  lazy val contravariantDerives = TypeClass("ContravariantDerives", *->*, extendsList = Seq(derives, codivide, divisible))
-  lazy val covariantDerives = TypeClass("CovariantDerives", *->*, extendsList = Seq(derives, coapplicative, applicative))
-  lazy val coapplicative = TypeClass("Coapplicative", *->*, extendsList = Seq(coapplicativeCodivide))
-  lazy val apply: TypeClass = TypeClass("Apply", *->*, extendsList = Seq(functor, applyDivide))
-  lazy val applicative = TypeClass("Applicative", *->*, extendsList = Seq(apply, applicativeDivisible))
+  lazy val derives = TypeClass("Derives", *->*, fromIso = false)
```

Comment:
Is there any reason `fromIso = false`?

Reply:
https://github.com/xuwei-k/scalaz/commit/25a150ff80e50015a8ff14f1508eccbd62e96f17

Reply:
I couldn't get it to work :-D

Reply:
:bowing_man: